docs/developer-docs/docs/build-an-agent/build-an-onchain-ai-agent/summary.md

```
@@ -0,0 +1,116 @@
+    }
+}
+```
+
+## Deployment and monitoring
+
+#### Automated Orders
+
+```bash
+just create-basic-order $THRESHOLD $CONDITION $PRICE_PAIR
+```
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)